src/app/ecosystem-explorer/project-form/actions/submitProject.ts

```diff
@@ -0,0 +1,49 @@
+'use server'
+
+import slugify from 'slugify'
+
+import { submitProjectToGithub } from '../services/github'
+import type {
+  EcosystemProjectFormDataWithoutLogo,
+  FormattedLogo,
+} from '../types'
+import { buildEcosystemMarkdownTemplate } from '../utils/buildEcosystemMarkdownTemplate'
+import { formatFormData } from '../utils/formatFormData'
+import { getFolderPaths } from '../utils/getFolderPaths'
+
+type SubmitProjectParams = {
+  formattedLogo: FormattedLogo
+  formDataWithoutLogo: EcosystemProjectFormDataWithoutLogo
+}
+
+export async function submitProject({
+  formattedLogo,
+  formDataWithoutLogo,
+}: SubmitProjectParams) {
+  const slug = slugify(formDataWithoutLogo.title, {
+    lower: true,
+    strict: true,
+  })
+  const logoName = `${slug}.${formattedLogo.format}`
```

Comment:
Filename?

Reply:
Let's change `formattedLogo` and `logoName` in accordance to `formatFile` function decisions.